content/docs/support/license.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "License"
+linkTitle: "License"
+weight: 11
+Description: >
+  Dell Container Storage Modules (CSM) License
+---
+
+The tech-preview releases of [Container Storage Modules](https://github.com/dell/csm) for Application Mobility and Encryption require a license. The following section details how to request a license.
+
+## Requesting a License
+1. Request a license using the [Container Storage Modules License Request](https://app.smartsheet.com/b/form/5e46fad643874d56b1f9cf4c9f3071fb) by providing the following details:
+- **Full Name**: Full name of the person requesting the license
+- **Email Address**: The license will be emailed to this email address
+- **Company / Organization**: Company or organization where the license will be used
+- **License Type**: Select either *Application Mobility* or *Encryption*, depending on the CSM module that will be used with the license
+- **List of kube-system namespace UIDs**: The license will only function on the provided list of Kubernetes clusters. Find the UID of the kube-system namespace using `kubectl get ns kube-system -o yaml` or similar `oc` command.
```

Comment:
I would only add, that it should be a comma-separated list - would be easier for us if we decide to automate/pre-process the requests. It would also add clarity for the user as to how to format the list.

Reply:
Fixed, mentioned it should be provided as a comma separated list